### PR TITLE
Cyborgs can now open task dispenser UI

### PR DESCRIPTION
**Pull request by JamieD1**
Cyborgs can now open task dispenser UI
:cl:  
rscadd: Cyborgs can now open task dispenser UI
/:cl:

https://user-images.githubusercontent.com/993128/165187021-47abf457-76cc-4e70-90cf-206c59d57d3a.mp4

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Why?

**Comment by JamieD1** [comment]
Will change this later

**Comment by ToasterBiome** [comment]
> Will change this later

What is point of PR

**Comment by JamieD1** [comment]
https://user-images.githubusercontent.com/993128/165186974-74819c09-ab26-446c-9dc2-454c47ba6253.mp4

File: code/game/objects/structures/tank_dispenser.dm [hunk 1]
```diff
@@ -90,13 +94,19 @@
 		if("plasma")
 			var/obj/item/tank/internals/plasma/tank = locate() in src
 			if(tank && Adjacent(usr))
-				usr.put_in_hands(tank)
+				if(iscyborg(usr))
+					usr.dropItemToGround(tank)
```
Comment (ToasterBiome) [inline code/game/objects/structures/tank_dispenser.dm]:
```suggestion
					usr.transferItemToLoc(tank, src.loc)
```

File: code/game/objects/structures/tank_dispenser.dm [hunk 2]
```diff
@@ -90,13 +94,19 @@
...
+				else
+					usr.put_in_hands(tank)
 				plasmatanks--
 			. = TRUE
 		if("oxygen")
 			var/obj/item/tank/internals/oxygen/tank = locate() in src
 			if(tank && Adjacent(usr))
-				usr.put_in_hands(tank)
+				if(iscyborg(usr))
+					usr.dropItemToGround(tank)
```
Comment (ToasterBiome) [inline code/game/objects/structures/tank_dispenser.dm]:
```suggestion
					usr.transferItemToLoc(tank,src.loc)
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]
them dropping under the silicon is annoying, make them drop from the dispenser (like a vending machine)

**Pull request merged**